apps/mobile/src/hooks/useNotificationGTWPermissions.ts

```diff
@@ -0,0 +1,22 @@
+
+import { useCallback } from 'react'
+import { useAppSelector } from '../store/hooks'
+import { useDefinedActiveSafe } from '../store/hooks/activeSafe'
+import { RootState } from '../store'
+import { selectSigners } from '../store/signersSlice'
+import { NOTIFICATION_ACCOUNT_TYPE } from '../store/constants'
+import { selectSafeInfo } from '../store/safesSlice'
+
+export function useNotificationGTWPermissions() {
+    const appSigners = useAppSelector(selectSigners)
+    const activeSafe = useDefinedActiveSafe()
+    const activeSafeInfo = useAppSelector((state: RootState) => selectSafeInfo(state, activeSafe.address))
+    const ownerFound = activeSafeInfo.SafeInfo.owners.find((owner) => appSigners[owner.value]) ?? null
+    const accountType = ownerFound ? NOTIFICATION_ACCOUNT_TYPE.OWNER : NOTIFICATION_ACCOUNT_TYPE.REGULAR
+
+    const getAccountType = useCallback(() => {
+        return accountType
+    }, [accountType])
```

Comment:
wouldn't it make more sense to have the activeSafeInfo.SafeInfo.owners as dependency here and do the onwerFound and accountType calculation inside?

Right now the func will recalculate whenever something in the safe changes.